dualscreeninfo/android/src/main/kotlin/com/microsoft/reactnativedualscreen/dualscreen/DualScreenInfo.kt

```diff
@@ -0,0 +1,138 @@
+package com.microsoft.reactnativedualscreen.dualscreen
+
+import android.content.Context
+import android.graphics.Rect
+import android.view.View
+import android.view.WindowManager
+import com.facebook.react.bridge.*
+import com.facebook.react.bridge.Arguments.createMap
+import com.facebook.react.modules.core.DeviceEventManagerModule.RCTDeviceEventEmitter
+import com.microsoft.device.display.DisplayMask
+
+const val HINGE_WIDTH_KEY = "hingeWidth"
+const val IS_DUALSCREEN_DEVICE_KEY = "isDualScreenDevice"
+const val FEATURE_NAME = "com.microsoft.device.display.displaymask"
+
+class DualScreenInfo constructor(context: ReactApplicationContext) : ReactContextBaseJavaModule(context) {
+    private val isDualScreenDevice: Boolean
+        get() =  reactApplicationContext.packageManager.hasSystemFeature(FEATURE_NAME)
+	private val mDisplayMask: DisplayMask
+		get() = DisplayMask.fromResourcesRectApproximation(currentActivity)
```

Comment:
>DisplayMask [](start = 10, length = 11)

If this code runs on a device that doesn't have the displayMask feature, what will happen?  It should gracefully fallback to returning info about the single screen.